src/spec-configuration/httpOCIRegistry.ts

```diff
@@ -198,6 +220,34 @@ async function getCredential(params: CommonParams, ociRef: OCIRef | OCICollectio
 	return;
 }
 
+async function getCredentialFromHelper(params: CommonParams, registry: string, credHelperName: string) : Promise<{ base64EncodedCredential: string | undefined; refreshToken: string | undefined } | undefined>{
+	const { output } = params;
+
+	const { stdout } = await runCommandNoPty({
```

Comment:
This would throw an error if the exit code is not zero. Should we handle that by returning undefined here?